### ISSUE TITLE
Bug when disabling windows and getting entries values on macOS

### ISSUE THREAD

**Issue by emersion** [andlabs/libui#196]
How to reproduce:
* Create a window with entries
* Add a button that: disables the whole window and after prints the entries value
* Run this, enter something in entries, remove focus from these entries (by clicking around), click the button
* Values printed are empty

A way to workaround this is to get these values before disabling the window. Another workaround: get entries values, throw away the returned results, disable the whole window, and get again values.

**Comment by andlabs** [comment]
Yes, disabling and enabling entries on OS X is buggy, due to the fact that `NSTextField.enabled`'s operation is itself unclear. I wonder if I can just reject all edits in the delegate...

**Comment by andlabs** [comment]
That being said, see if the latest master fixes the problem at all?

**Comment by andlabs** [comment]
Can you also see if the problem goes away if you use an app bundle?
